### DISCUSSION TITLE
React源码怎么入手

### DISCUSSION THREAD

**Discussion by shengxj1** [shengxinjing/it-roadmap#8] [提问] [answered]
求推荐视频

**Comment by johanazhu** [comment, marked as answer]
卡颂 react 源码
https://react.iamkasong.com
当然，他还有付费的视频课程

**Reply by shengxinjing** [reply]
very good

**Comment by itisyxfan** [comment]
视频地址
https://ke.segmentfault.com/list?keyword=%E8%87%AA%E9%A1%B6%E5%90%91%E4%B8%8B

**Comment by linhexs** [comment]
建议先手写一个mini版的，再去读源码，这样会更好理解；推荐一下写的一篇文章 [Mini-React](https://juejin.cn/post/7023319961883901988)
